lib/mkl/wrappers.jl

```diff
@@ -20,6 +20,25 @@ end
 # BLAS
 #
 
+# level 1
+## copy
+for (fname, elty) in
+        ((:onemklDcopy,:Float64),
+         (:onemklScopy,:Float32),
+         (:onemklZcopy,:ComplexF64),
+         (:onemklCcopy,:ComplexF32))
+    @eval begin
+        function copy!(n::Integer,
+                       x::StridedArray{$elty},
```

Comment:
It would probably be better to use oneStridedArray here so that you can't invoke this functionality using non-oneAPI arrays.